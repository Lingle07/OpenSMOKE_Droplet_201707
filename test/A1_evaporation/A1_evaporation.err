/var/spool/torque/mom_priv/jobs/202346.certainty-fe.stanford.edu.SC: line 26: 21528 Segmentation fault      $PBS_O_WORKDIR/OpenSMOKEpp_MicrogravityDroplet.sh --input $PBS_O_IODIR/input.dic > $PBS_O_IODIR/log.$PBS_JOBID
