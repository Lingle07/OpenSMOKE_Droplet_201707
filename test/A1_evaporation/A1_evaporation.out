The master node of this job is compute-136-3.local
The working directory is /home/xyhuang/OpenSMOKE_Droplet/exe
This job runs on the following nodes:
compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3 compute-136-3
Running Intel-15 MVAPICH linked bounce with:  in directory /home/xyhuang/OpenSMOKE_Droplet/exe/examples/test/A1_evaporation
